.github/workflows/cla.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: "CLA Assistant"
         if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
         # Beta Release
```

Comment:
Do we still need the "Beta release" comment?

Reply:
I don't think so